GERTi/GERTi Client.lua

```diff
@@ -34,7 +48,7 @@ local function storeNeighbors(eventName, receivingModem, sendingModem, port, dis
     neighbors[neighborDex]["address"] = sendingModem
     neighbors[neighborDex]["port"] = tonumber(port)
     if package == nil then
-        --This is used for when a computer receives a new client's GERTiStart message. It stores a neighbor connection with a tier one lower than this computer's tier
+        --This is used for when a computer receives a new client's GERTiStart message. It stores a neighbor connection with a tier one higher than this computer's tier
```

Comment:
It should be "a tier one lower", because a higher number indicates a client further away from the gateway, which is a lower tier.